rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form-original.json

```diff
@@ -560,7 +560,11 @@
         "hint": "Date of onset of symptoms",
         "expanded": false,
         "min_date": "today-100y",
-        "max_date": "today"
+        "max_date": "today",
+        "v_required": {
+          "value": "true",
+          "err": "This field is required"
+        }
```

Comment:
This is makes the field required. This shouldn't be the case https://github.com/onaio/rdt-standard/issues/799#issue-813950999.

Reply:
yes, I have made this field is required again according to this comment https://github.com/onaio/rdt-standard/issues/799#issuecomment-787608309

Reply:
It's not meant to be required, could you verify that it's not required currently and see why it may be showing as required if so?

Reply:
I have checked in master currently it's not required.

Reply:
have you tried navigating to the next step to see if an error shows on that field?

Reply:
yeah, I have tried navigating to the next fields but it works fine. No error shows on that fields

Reply:
OK, you can close this